packages/react-charting/package.json

```diff
@@ -17,15 +17,16 @@
     "build": "just-scripts build",
     "bundle": "just-scripts bundle",
     "lint": "just-scripts lint",
-    "test": "just-scripts test",
+    "test": "cross-env TZ=UTC just-scripts test",
```

Comment:
does TZ=UTC change the time zone for other packages as well.

Reply:
I don't think so. Found this in the logs:
```
...
2024-02-05T08:08:24.7357941Z verb @fluentui/react-charting test |  > @fluentui/react-charting@5.19.21 test
2024-02-05T08:08:24.7358443Z verb @fluentui/react-charting test |  > cross-env TZ=UTC just-scripts test
...
2024-02-05T08:08:40.1172951Z verb @fluentui/react-monaco-editor test |  > @fluentui/react-monaco-editor@1.7.171 test
2024-02-05T08:08:40.1173859Z verb @fluentui/react-monaco-editor test |  > just-scripts test
...
2024-02-05T08:08:58.2219995Z verb @fluentui/react-docsite-components test |  > @fluentui/react-docsite-components@8.13.53 test
2024-02-05T08:08:58.2221137Z verb @fluentui/react-docsite-components test |  > just-scripts test
...
2024-02-05T08:09:12.9082014Z verb @fluentui/theming-designer test |  > @fluentui/theming-designer@1.0.0 test
2024-02-05T08:09:12.9082784Z verb @fluentui/theming-designer test |  > just-scripts test
...
2024-02-05T08:10:06.2250466Z verb @fluentui/public-docsite test |  > @fluentui/public-docsite@8.2.41 test
2024-02-05T08:10:06.2251061Z verb @fluentui/public-docsite test |  > just-scripts test
...
2024-02-05T08:10:10.5854905Z verb @fluentui/ssr-tests test |  > @fluentui/ssr-tests@1.0.0 test
2024-02-05T08:10:10.5855341Z verb @fluentui/ssr-tests test |  > just-scripts test
...
```

Reply:
Yes, the changes in env won't be visible outside the react-charting test process.